sdl2promela/stop_condition/translator.py

```diff
@@ -490,6 +490,61 @@ def _generate_exist(context: GenerateContext, expr: model.CallExpression):
     )
 
 
+def _get_queue_name(context: GenerateContext, expr: model.CallExpression):
+    if len(expr.parameters) != 1:
+        raise TranslateException("Function 'empty' requires one parameter")
```

Comment:
Why empty? How is it related to the name of the method?

Reply:
This is also used in queue_length method....

Reply:
Ou, this is a Stop Condition which I've implemented, but it needs proper tests...